nextflow_schema.json

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "https://raw.githubusercontent.com/phac-nml/iridanextexample/main/nextflow_schema.json",
-    "title": "phac-nml/iridanextexample pipeline parameters",
+    "$id": "https://raw.githubusercontent.com/phac-nml/clustersplitter/main/nextflow_schema.json",
```

Comment:
1. In this config file, can you remove the parameter to set `av_html` and have it fixed to use the one in the `assets/` directory, similar to gasclustering?

Reply:
Updated in 99c2d94756d310b3f7eab374bb70a5855b2202ac